src/sentry/search/eap/resolver.py

```diff
@@ -450,8 +450,23 @@ def resolve_term(
 
         if value == "" and context_definition is None:
             exists_filter = TraceItemFilter(
-                exists_filter=ExistsFilter(
-                    key=resolved_column.proto_definition,
+                and_filter=AndFilter(
+                    filters=[
+                        TraceItemFilter(
+                            exists_filter=ExistsFilter(
+                                key=resolved_column.proto_definition,
+                            )
+                        ),
```

Comment:
Does this need to be negated for equality comparisons?

Reply:
ah good call, need to rewrite the negation handling